README.md

```diff
@@ -16,16 +16,13 @@ use Larium\Framework\Provider\ContainerProvider;
 require_once __DIR__ . '/../vendor/autoload.php';
 
 (function () {
-    /** @var ContainerProvider */
-    $containerProvider
```

Comment:
This line shows that somehow you have to setup a `containerProvider`

Reply:
I will re-open pull that contains only the typo fix (dot has to be replaced with greek question mark) thanks for the information

Reply:
> This line shows that somehow you have to setup a `containerProvider`

I reverted and I replaced only the dot with the question mark. Is it a typo, isn't it?